includes/wp-cli/class-orchestrate-runner.php

```diff
@@ -84,7 +84,13 @@ public function run_event( $args, $assoc_args ) {
 		$run = \Automattic\WP\Cron_Control\run_event( $timestamp, $action, $instance );
 
 		if ( is_wp_error( $run ) ) {
-			\WP_CLI::error( $run->get_error_message() );
+			$error_data = $run->get_error_data();
+
+			if ( isset( $error_data['status'] ) && 404 === $error_data['status'] ) {
+				\WP_CLI::success( $run->get_error_message() );
```

Comment:
Should this be a warning + early return?

Reply:
A warning is more appropriate, but the early return isn't needed as nothing else should happen in this function after the message is output.

Reply:
👍 switched to warning in 381d38da09bdaa4587f8058b4f56ab5a4243c94e